source/ui.py

```diff
@@ -123,6 +131,12 @@ def browseableMessage(message: str, title: Optional[str] = None, isHtml: bool =
 		return
 	d.add("title", title)
 	d.add("message", message)

```

Comment:
Adding this, although I can not think of circumstances where MSHTML would be unable to copy to clipboard, in the contexts where we allow it to run.

```suggestion
	d.add("message", message)
	# Translators: A notice to the user that the copy operation succeeded.
	d.add("copySuccessfulAlertText", _("Text copied."))
	# Translators: Notice to the user that the copy operation failed.
	d.add("copyFailedAlertText", _("Couldn't copy to clipboard."))
```